aws-history: add page (#8805)
tldr-pages/tldr: pages/common/aws-history.md

# aws history

> > Commands to interact with the history of AWS CLI commands ran over time (the record of history of AWS CLI commands must be enabled).

Reviewer: ```suggestion
> Print the command-line history for AWS CLI commands (the record of history of AWS CLI commands must be enabled).
```

Author: Thanks for all suggestions! 😄

> More information: <https://docs.aws.amazon.com/cli/latest/reference/history/>.

> - Get a list of previously run commands and their command ids:

Reviewer: Perhaps this better
```suggestion
- List commands history with command IDs:
```

`aws history list`

> - Get the various events related to running a specific CLI command with its command id:

Reviewer: ```suggestion
- Display events related to a specific command given a command ID:
```

> `aws history show {{1affe293-ed81-451c-a044-ea99223a93bf}}`

Reviewer: Perhaps a placeholder is better in this example
```suggestion
`aws history show {{command_id}}`
```
